backend/zacniucit/settings.py

```diff
@@ -169,15 +178,56 @@
 
 AUTH_USER_MODEL = 'accounts.CustomUser'
 
+AUTHENTICATION_BACKENDS = [
+    "graphql_auth.backends.GraphQLAuthBackend",
+    "django.contrib.auth.backends.ModelBackend",
+]
+
+GRAPHQL_JWT = {
+    "JWT_VERIFY_EXPIRATION": True,
+    "JWT_EXPIRATION_DELTA": timedelta(minutes=5),
+    "JWT_REFRESH_EXPIRATION_DELTA": timedelta(days=7),
+    "JWT_LONG_RUNNING_REFRESH_TOKEN": True,
+    "JWT_ALLOW_ANY_CLASSES": [
+        "accounts.graphql.mutations.CustomRegister",
+        "graphql_auth.mutations.PasswordSet",
+        "graphql_auth.mutations.RevokeToken",
+        "graphql_auth.mutations.ObtainJSONWebToken",
+        "graphql_auth.mutations.RefreshToken",
+    ],
+}
+
+
+GRAPHQL_AUTH = {
+    "LOGIN_ALLOWED_FIELDS": ['email'],
+    "REGISTER_MUTATION_FIELDS": {
+        "email": "String"
+    },
+    "ALLOW_LOGIN_NOT_VERIFIED": False,
+    "ALLOW_PASSWORDLESS_REGISTRATION": True,
+
+    # We don't want to automatically send emails, because we are using sendGrid templates
+    "SEND_PASSWORD_SET_EMAIL": False,
+    "SEND_ACTIVATION_EMAIL": False,
+}
+
 
 #########
 # Email #
 #########
 
 EMAIL_BACKEND = "anymail.backends.sendgrid.EmailBackend"
 
-ANYMAIL = {
-    "SENDGRID_API_KEY": config('SENDGRID_API_KEY', default='', cast=str)
-}
+ANYMAIL = {"SENDGRID_API_KEY": config("SENDGRID_API_KEY", default="", cast=str)}
+
+DEFAULT_FROM_EMAIL = config("DEFAULT_FROM_EMAIL", default="notifications@cesko.digital", cast=str)
+
+SET_EMAIL_TEMPLATE_ID = config('SET_EMAIL_TEMPLATE_ID', default="d-e58beb380bed4cb7afb8a9e456263a8d", cast=str)
+
+
+############
+# Frontend #
+############
 
-DEFAULT_FROM_EMAIL = config('DEFAULT_FROM_EMAIL', default='notifications@cesko.digital', cast=str)
\ No newline at end of file
+# TODO Don't know what is fronted url, let's change it when we know
+BASE_FRONTED_URL = config("BASE_FRONTED_URL", default="https://zacniucit.cz")
```

Comment:
Davide, nepustil jsi nahodou black i na settings.py soubor?

Tenhle soubor, stejne jako migrace zamerne blackem ignorujeme, protoze jsou budto cele nebo castecne generovane Djangem, viz: https://github.com/cesko-digital/zacni-ucit/blob/master/pyproject.toml#L4

Drzme se prosim toho, jinak se budeme donekonecna pretahovat jak ma byt konkretni cast kodu naformatovana (a zde konkretne se pak ztraci i to co se vlastne v souvislosti s GraphQL PR meni)

Reply:
Tohle jsem nevěděl a právě jsem spustil black na settings.py :/ Mám to nějak předělávat. Nebo to už prostě jenom nemám dělat příště?

Reply:
Vrat to prosim do puvodniho stavu, a pridej tam jen to co souvisi s PR. Dikes.